### ISSUE TITLE
The `SortedSet` class has been extracted from the `set` library.You must use the `sorted_set` gem or other alternatives.

### ISSUE THREAD

**Issue by hicsuntdragons** [rccavalcanti/fyodor#4]
Hello!

Im trying to use fyodor to convert 'My Clippings.txt' in the described way on the readme. I have configured the `fyodor.toml` file to brazilian portuguese.

I run: 
```sh
fyodor My\ Clippings.txt output.md
```

and get this output:
```sh
Using config at /home/juliac/.config/fyodor.toml

The `SortedSet` class has been extracted from the `set` library.You must use the `sorted_set` gem or other alternatives.
```

and no output is provided. Any ideas?

**Comment by rc2dev** [comment]
Hi!

Can you check your ruby version with `ruby -v`?

**Comment by hicsuntdragons** [comment]
Sure!
```sh
$ ruby -v
ruby 3.0.1p64 (2021-04-05 revision 0fb782ee38) [x86_64-linux]
```

**Comment by rc2dev** [comment]
Thank you for reporting!

This is an error introduced by Ruby 3. While this is not fixed, you can use Ruby 2.7.

**Comment by camr12** [comment]
Same error using Ruby 2.7.2.1
